### PR TITLE
1151 author copyedit email

### LINKED ISSUE #1151: Author Copyedit Email
**Describe the bug**
The author copyedit email does not contain the correct URL.

**Janeway version**
1.3.4

**Expected behavior**
This should link

### PR DESCRIPTION
**Pull request by ajrbyers**
Closes #1151

### CODE REVIEW

File: src/copyediting/logic.py [hunk 1]
```diff
@@ -105,36 +110,44 @@ def handle_file_post(request, copyedit):
         return errors
 
 
-def request_author_review(copyedit, article, request):
+def setup_author_review(copyedit, article, request):
```
Comment (mauromsl) [inline src/copyediting/logic.py]: No need for the request to be passed here anymore, in fact, I think we can do away with this function entirely.

File: src/copyediting/views.py [hunk 2]
```diff
@@ -447,13 +465,89 @@ def editor_review(request, article_id, copyedit_id):
...
+            )
+        )
+
+    template = 'copyediting/request_author_copyedit.html'
+    context = {
+        'article': article,
+        'copyedit': copyedit,
+        'author_review': author_review,
+        'email_content': email_content,
+        'email_subject': email_subject
```
Comment (mauromsl) [inline src/copyediting/views.py]: trailing comma

File: src/copyediting/migrations/0005_auto_20190523_1758.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+        for value in values:
+            translations = SettingValueTranslation.objects.filter(
+                master_id=value.pk
+            )
+
+            for translation in translations:
+                value = translation.value
+                translation.value = value.replace(
+                    '{{journal.site_url}}{{ url }}',
```
Comment (mauromsl) [inline src/copyediting/migrations/0005_auto_20190523_1758.py]: the default value that you replaced below was:
`{{request.journal.site_url}}{{ url }}`
We need to handle that as well

File: src/copyediting/migrations/0005_auto_20190523_1758.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                    "{% journal_url 'author_copyedit' article.pk author_review.pk %}"
```
Comment (mauromsl) [inline src/copyediting/migrations/0005_auto_20190523_1758.py]: trailing comma

File: src/copyediting/migrations/0005_auto_20190523_1758.py (hunk 3 above)
Comment (ajrbyers) [reply src/copyediting/migrations/0005_auto_20190523_1758.py]: Added.

**Comment by ajrbyers** [comment]
Changes made.

**Pull request merged**